docs/user/contributing.md

```diff
@@ -125,6 +126,6 @@ If one wants to account for these factors, one must introduce them manually by c
   
 
 #### Tips & tricks
-* You can find a wealth of pytorch debugging tricks at [their contributing page](https://github.com/pytorch/pytorch/blob/master/CONTRIBUTING.md).
+* You can find a wealth of pytorch debugging tricks at [their contributing page](https://tinyurl.com/pytorch-contributing).
```

Comment:
old link wouldn't render correctly after `make html`

Reply:
yep, its their main guarantee
so, not "absolutely guaranteed", but no worse than with a direct github link